it/src/test/java/com/linecorp/centraldogma/it/updater/CentralDogmaBeanTest.java

```diff
@@ -87,6 +93,23 @@ public void test() throws Exception {
 
         assertThat(property.getBar()).isEqualTo("Y");
         assertThat(property.getQux()).containsExactly("0", "1");
+        assertThat(property.getRevision()).isEqualTo(new Revision("2"));
+        assertThat(called[0]).isEqualTo(1);
+
+        property.closeWatcher();
```

Comment:
Could you do this in the `overrideSettings()` as well please?

Reply:
I'm reaaaaaaaaaaaaaaaaaally sorry for this ambiguous word. I was just asking you to insert only the `property.closeWatcher()` to the `overrideSettings()`.
When there's no way to close the watcher, it was closed with an exception some times after  the `property` is created, and it polluted the logs from gradle build. That is why I asked you to close it explicitly. :-)

Reply:
I see, thanks for the explaination, fixed :)